docs/_includes/calendar.html

```diff
@@ -262,22 +564,181 @@ <h3>Interactive Calendar</h3>
+            <table>
+                <caption>August 2023</caption>
+                <thead>
+                    <tr>
+                        <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="2"></td>
+                        <td><button disabled>1</button></td>
+                        <td><button disabled>2</button></td>
+                        <td><button disabled>3</button></td>
+                        <td><button disabled>4</button></td>
+                        <td><button disabled>5</button></td>
+                    </tr>
+                    <tr>
+                        <td><button disabled>6</button></td>
+                        <td><button disabled>7</button></td>
+                        <td><button disabled>8</button></td>
+                        <td><button disabled>9</button></td>
+                        <td><button disabled>10</button></td>
+                        <td><button disabled>11</button></td>
+                        <td><button disabled>12</button></td>
+                    </tr>
+                    <tr>
+                        <td><button disabled>13</button></td>
+                        <td><button aria-current="date">14</button></td>
+                        <td><button>15</button></td>
+                        <td><button>16</button></td>
+                        <td class="calendar__cell--selected calendar__range calendar__range--start">
+                            <button aria-label="17 - start of range" aria-pressed="true">17</button>
+                        </td>
+                        <td class="calendar__range">
+                            <button aria-label="18 - in range">18</button>
+                        </td>
+                        <td class="calendar__range">
+                            <button aria-label="19 - in range">19</button>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><button>20</button></td>
+                        <td class="calendar__range">
+                            <button aria-label="21 - in range">21</button>
+                        </td>
+                        <td class="calendar__range">
+                            <button aria-label="22 - in range">22</button>
+                        </td>
+                        <td class="calendar__range">
+                            <button aria-label="23 - in range">23</button>
+                        </td>
+                        <td class="calendar__range">
+                            <button aria-label="24 - in range">24</button>
+                        </td>
+                        <td class="calendar__cell--selected calendar__range calendar__range--end">
+                            <button aria-label="25 - end of range" aria-pressed="true">25<</span>
```

Comment:
Don't know how that extra `<` got in there before the last span, just a small typo ;)